8 Ball/Modules/AnswerList/AnswerListRouter.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  AnswerListRouter.swift
+//  8 Ball
+//
+//  Created by Roman Topchii on 20.02.2022.
+//
+
+import UIKit
+
+enum AnswerListDestination: RoutingDestinationBase {
+    case editAnswer(answer: Answer, delegate: AnswerListListener)
+    case addAnswer(delegate: AnswerListListener)
+}
+
+class AnswerListRouter: Router {
+    typealias RoutingDestination = AnswerListDestination
+    
+    private unowned var viewController: UIViewController!
```

Comment:
```suggestion
    private let viewController: UIViewController
```

Reply:
если не сделать сcылку слабой, то получиться замыкание сильных ссылок  router->view controller, viewController->router

```suggestion
    private let viewController: UIViewController
```

Reply:
Да, ты прав, не заметила что друг на друга ссылаются